### ISSUE TITLE
CLOSED BY MISTAKE [I guess] "Title bar does not appear and I would like to see the title bar"

### ISSUE THREAD

**Issue by TheBlackSun** [Milly/compact_menu_2#19]
-> https://github.com/Milly/compact_menu_2/issues/17

Dunno why he closed it, he didn't get my Point. It's just a BUG that causes the Titlebar to look like he would like to have it, but it will reset/fix itself upon Browser Restart. Still, Button to chose either 'Mode' optional and intentional isn't implemented yet. Would be a nice Feature!

**Comment by Milly** [comment]
Please tell us the following: OS and version, Application(Firefox, Thunderbird, SeaMonkey) and version, Other add-ons you installed

**Comment by derwan** [comment]
Win XP Firefox 6.0 and many extensions (is there a way to list many extensions?)

**Comment by Milly** [comment]
If only install Compact-Menu-2, does the problem occur?
Otherwise, I don't know what that conflicts with any add-ons.

**Comment by derwan** [comment]
Yes - When there is only your extension working, I still can not see the titlebar.

**Comment by Milly** [comment]
Use new profile?
Can you upload screen-shot?

**Comment by derwan** [comment]
Hi - I do not see here how I can upload an image here but I sent it to you by email. In any case if the option is hard to implement, I can use it this current way too. In any case compact menu's aim is as minimal as it is possible and not to take from the screen estate - so if the option is hard to apply, that would be fine too.
